sdk/textanalytics/ai-text-analytics/samples/javascript/analyzeSentimentWithOpinionMining.js

```diff
@@ -3,11 +3,11 @@
 
 /**
  * This sample demonstrates how to analyze sentiment on a more granular level,
- * mining individual opinions from reviews (also known as aspect-based
```

Comment:
I think `opinions` is better here. it introduces the feature `Opinion mining` and it means the service extracted `opinions` which include `assessment's` and `targets`